pyfftw/pyfftw.pyx

```diff
@@ -41,9 +41,16 @@ from libc.stdlib cimport calloc, malloc, free
 from libc.stdint cimport intptr_t, int64_t
 from libc cimport limits
 
+import platform
 import warnings
 import threading
 
+# the fftw .dll (3.3.5) does not export fftw*version, fftw*_cc and
+# fftw*_codelet_optim etc., so for Windows the version is hard-coded to
+# 3.3.5 and others as 'NOT_AVAILABLE'
+_ON_WINDOWS = platform.system() == 'Windows'
+_WINDOWS_FFTW_VERSION = '3.3.5'
```

Comment:
@paugier 3.3.5 is the latest FFTW Windows DLL provided via fftw.org so I just chose 3.3.5. Of course people can build their own DLL with different version of fftw, so perhaps it is better to list it as `NOT_AVAILABLE` altogether as this reflects the fact that it is not available from the underlying fftw library directly. What do you think?

Reply:
I would prefer not to give a potentially wrong information, so I think `"NOT_AVAILABLE"` make sense when the data is not available.

This lack of data on Windows is annoying. It means that  code like `if pyfftw.fftw_version_tuple > (3, 0, 0):` cannot be used in practice because it could be wrong on Windows.

However, is there a practical reason to use such checks?

Reply:
@paugier I agree this is a bit annoying. I have emailed the authors of fftw about this, as I think this is an issue upstream (I may be wrong). Unless they provide a fftw .dll with those symbols exported, we can't reliably query the fftw version.

In some cases checking the fftw version may be helpful if the user depends on new features or bug fixes that are only available on some later versions. The compiler flag may be also useful in knowing whether fftw is complied with certain features.